certgen/certProcessor/src/main/java/org/incredible/certProcessor/signature/SignatureHelper.java

```diff
@@ -80,7 +80,7 @@ public boolean verifySignature(JsonNode rootNode)
         signReq.put(JsonKey.ENTITY, rootNode);
         boolean result = false;
         CloseableHttpClient client = HttpClients.createDefault();
-        HttpPost httpPost = new HttpPost(properties.get(JsonKey.SIGN_VERIFY_URL));
+        HttpPost httpPost = new HttpPost(encServiceUrl.concat("/" + JsonKey.VERIFY));
```

Comment:
Keep this and sign endpoints as final private strings.

Reply:
done